lua/mason-lspconfig/mappings/filetype.lua

```diff
@@ -242,8 +242,9 @@ return {
   ["yaml.ansible"] = { "ansiblels" },
   ["yaml.docker-compose"] = { "docker_compose_language_service", "yamlls" },
   ["yaml.gitlab"] = { "gitlab_ci_ls", "yamlls" },
+  ["yaml.github"] = { "gh_actions_ls", "yamlls" },
```

Comment:
```suggestion
  ["yaml.github"] = { "gh_actions_ls", "yamlls" },
```

now that the filetype has been changed upstream to `yaml` we should also update it here

Reply:
update how? the suggestion contains no changes.

Reply:
my bad, original answer updated. The LS needs to be registered for the regular `yaml` filetype now.

Reply:
Wouldn't that be contrary to the advice given [here](https://github.com/neovim/nvim-lspconfig/pull/3551/files#diff-ea3e046455fe04df9ed4b3b36f255b8e1f46e119006f0e18f482489fa514b763R26)?

Reply:
@disrupted 👆 Do I still need to adjust this part?

Reply:
yes, since it was changed upstream it needs to be changed here as well https://github.com/neovim/nvim-lspconfig/commit/0c24b48da1b70bbf06075c51fa0e1c87d1337c11

Reply:
ok changed in f9b5134ebddbf0529cf198954530da75d70cdf98